packages/devtools_app/benchmark/README.md

```diff
@@ -51,14 +61,24 @@ a new one for a new screen. Follow existing examples in that directory for guida
 
 ## Comparing two benchmark test runs
 
-In order to compare two different benchmark runs, you first need to run the benchmark
-tests and save the results to a file:
-```sh
-dart run benchmark/scripts/run_benchmarks.dart --save-to-file=baseline.json
-dart run benchmark/scripts/run_benchmarks.dart --save-to-file=test.json
-```
-
-Then, to compare the benchmarks and calculate deltas, run:
-```sh
-dart run benchmark/scripts/compare_benchmarks.dart baseline_file.json test_file.json
-```
+There are two different ways to calculate the delta between two benchmark test runs.
+Choose the one that best fits your use case.
```

Comment:
Remove 'Choose the one that best fits your use case.'
It does not add any information.